config/main.py

```diff
@@ -1363,6 +1363,47 @@ def apply_patch(ctx, patch_file_path, format, dry_run, ignore_non_yang_tables, i
             patch = jsonpatch.JsonPatch(patch_as_json)
 
         config_format = ConfigFormat[format.upper()]
+        if multi_asic.is_multi_asic():
+            # Initialize a dictionary to hold changes categorized by ASIC
+            changes_by_asic = {}
+
+            # Function to extract ASIC identifier from the change path
+            def extract_asic_id(path):
+                start = path.find("/") + 1
+                end = path.find("/", start)
+                return path[start:end], path[end:]  # Also return the modified path without ASIC ID
+
+            # Function to apply patch for a single ASIC.
+            def apply_patch_for_asic(asic_changes):
+                print(asic_changes)
+                asic_id, changes = asic_changes
+
+                # Replace localhost to empty string which is db definition of Host
+                if asic_id.lower() == "localhost":
+                    asic_id = ""
+
+                # Call apply_patch with the ASIC-specific changes and predefined parameters
+                GenericUpdater(namespace=asic_id).apply_patch(jsonpatch.JsonPatch(changes), config_format, verbose, dry_run, ignore_non_yang_tables, ignore_path)
+
+            print("Patch: {}".format(patch))
+            # Iterate over each change in the JSON Patch
+            for change in patch:
+                asic_id, modified_path = extract_asic_id(change["path"])
```

Comment:
What happens if "path" not in change?

Reply:
According to RFC: [https://www.rfc-editor.org/rfc/rfc6902#section-4](https://www.rfc-editor.org/rfc/rfc6902#section-4), operation objects MUST have exactly one "path" member.